### ISSUE TITLE
`tradis_merge_plots` cannot find `tradisfind`

### ISSUE THREAD

**Issue by cerebis** [sanger-pathogens/Bio-Tradis#133]
When using the Docker image within Singularity, invoking tradis_merge_plots produces the error:

```shell
sh: 1: tradisfind: not found
```

This is not surprising as I cannot find an instance of `tradisfind` within the Bio-Tradis repository.

Is it possible that this tool was mistakenly left behind within your dev/execution environment?

**Comment by cerebis** [comment]
Looking further, it appears these scripts are also relying on LSF calls (`bsub`)-- so I suspect not really intended for public consumption.

**Comment by lbarquist** [comment]
Yeah, this wasn't part of the original distribution and looks like it's been added for internal use at Sanger where sequencing might be split up across multiple lanes/runs. If you need this functionality, the script looks like it could be modified pretty easily -- or cat-ing together the fastq files would probably work just as well. This script isn't really needed to just run a TraDIS analysis.

**Comment by cerebis** [comment]
Ok thanks for the info. I did end up merging the separate plot files in
pandas through summmation at each coordinate.  I suspect I could have done
this at an earlier stage and avoided a bit of scripting.

Regards
Matt DeMaere
